### DISCUSSION TITLE
How to make merged text input field?

### DISCUSSION THREAD

**Discussion by skorphil** [chakra-ui/chakra-ui#8231] [Q&A] [answered]
Hi, how can i achieve merged input in Chakra?
![Screenshot 2024-01-23 at 21 37 18](https://github.com/chakra-ui/chakra-ui/assets/6762009/054028e5-48c8-48dc-bc5e-78f7a0bae8b4)

**Comment by segunadebayo** [comment, marked as answer]
Use `InputGroup` with attached addons or `startElement` / `endElement`.
